advocacy_docs/supported-open-source/postgresql/uninstalling.mdx

```diff
@@ -51,15 +52,21 @@ These instructions apply if you used EDB's interactive Mac installer to install
 
 To uninstall PostgreSQL on a Mac system:
 
-1.  Assume the identity of an operating system superuser, and navigate into the folder where the uninstaller resides:
+1.  Assume the identity of an operating system superuser.
```

Comment:
I'm still rather unsure about this step - if it's actually necessary, I would expect an example - AFAIK, this is somewhat more involved on macs than on most Linux distros.

We should try to get more info here.

Reply:
@josh-heyer, 
sudo /Library/PostgreSQL/13/uninstall-postgresql.app/Contents/MacOS/installbuilder.sh  

launches the uninstaller in GUI mode on Mac

<img width="422" alt="Screenshot 2023-09-01 at 4 10 54 PM" src="https://github.com/EnterpriseDB/docs/assets/59045594/470e3ddb-e518-49a8-b770-deaac9902693">

Reply:
@nidhibhammar does that mean step 1 and 2 aren't necessary?

Reply:
I have combined both steps and written as one step

Reply:
Note that this also executes a shell script instead of trying to launch the app directly. Not sure what's in the script, but perhaps this is why @kiisodj was encountering issues. If the shell script is the recommended way of launching the uninstaller, let's update the docs ASAP!

Reply:
@nidhibhammar can you confirm that the shell script is the recommended way of launching the uninstaller?

Reply:
YES , that's the right way to launch the uninstaller on MAC 

sudo /Library/PostgreSQL/13/uninstall-postgresql.app/Contents/MacOS/installbuilder.sh